__tests__/Calendar.react-test.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import moment from 'moment';
+
+import Calendar from '../src/js/components/calendar';
+import { DEFAULT_TIMESLOTS } from '../src/js/constants/day';
+
+describe('Render tests', () => {
+  test('Renders Correctly.', () => {
+    const tree = renderer.create(
+      <Calendar
+        initialDate = { moment().format() }
+        timeslots = { DEFAULT_TIMESLOTS }
```

Comment:
It is. Default Timeslots will only be passed down by parent component - children component don't auto-assume them.